src/kOS.Safe/Screen/ScreenBuffer.cs

```diff
@@ -48,15 +60,18 @@ public int AbsoluteCursorRow
         public ScreenBuffer()
         {
             buffer = new List<char[]>();
+            notifyees = new List<ResizeNotifier>();
+            
             subBuffers = new List<SubBuffer>();
 
-            RowCount = MAX_ROWS;
-            ColumnCount = MAX_COLUMNS;
+            RowCount = DEFAULT_ROWS;
+            ColumnCount = DEFAULT_COLUMNS;
             InitializeBuffer();
         }
 
         private void InitializeBuffer()
         {
+            buffer.Clear();
             AddNewBufferLines(RowCount);
```

Comment:
It used to be the case that InitializeBuffer was only called at the start of the constructor and never again.  Now it's getting called at other times too, thus the need to explicitly Clear.